virtualizarr/zarr.py

```diff
@@ -198,6 +191,23 @@ def _v3_codec_pipeline(self) -> Any:
 
         return codec_pipeline
 
+    def serializer(self) -> Any:
```

Comment:
This function could stand to get reviewed and tested. It looks like it was meant to be just a sketch of what a serializer could look like

Reply:
If this passes the tests we can perhaps merge and deal with this in the follow-up PR that refactors `ZArray` away entirely?

Reply:
Yeah exactly.